ui/components/app/modals/account-details-modal/account-details-modal.component.js

```diff
@@ -71,6 +71,12 @@ export default class AccountDetailsModal extends Component {
       exportPrivateKeyFeatureEnabled = false;
     }
 
+    ///: BEGIN:ONLY_INCLUDE_IN(snaps)
+    if (keyring?.type.includes('Snap')) {
+      exportPrivateKeyFeatureEnabled = false;
+    }
```

Comment:
Can we use a constant value to compare this condition instead of an `includes`? For example,

```ts
keyring?.type === KeyringType.Snap
```

Reply:
created new util `isAbleToExportAccount`